libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -224,6 +224,14 @@ type CurrencyTransaction<T extends TransactionCommon> = {
   ) => any;
 };
 
+export type AccountTestData<T extends TransactionCommon> = {
+  raw: AccountRaw;
+  implementations?: string[];
+  FIXME_tests?: Array<string | RegExp>;
```

Comment:
Do we still need the `FIXME` prefix ?

Reply:
I don't know. This is a simple copy paste of legacy integration tests.